docs/cluster_spec.md

```diff
@@ -446,6 +446,18 @@ Will result in the flag `--feature-gates=Accelerators=true,AllowExtTrafficLocalE
 
 NOTE: Feature gate `ExperimentalCriticalPodAnnotation` is enabled by default because some critical components like `kube-proxy` depend on its presence.
 
+Some feature gates (e.g. `PodShareProcessNamespace`) also require the `featureGates` setting to be used on the kubeAPIServer:
```

Comment:
Might be better to point out that feature gates may require being enabled on one or more k8s components, and not call out just the APIServer.  Also include a link to the k8s  feature gate page for more information. https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Reply:
Sure, that makes sense, I just wish there was some documentation about where the feature gates need to be enabled - I only found out because I asked on slack and someone helpfully replied.

Reply:
I've updated the PR to reflect your feedback